pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceUsageTransportManager.java

```diff
@@ -159,9 +171,11 @@ public void received(Reader<byte[]> reader, Message<byte[]> msg) {
     private final Map<String, ResourceUsageConsumer>
             consumerMap = new ConcurrentHashMap<String, ResourceUsageConsumer>();
 
+    private long staleMessageCount = 0;
```

Comment:
This variable doesn't seem to be used anywhere?

Reply:
I added the variable with the intention of adding a unit test for it. But could not figure out a easy way to do so. I am open to ideas if you have suggestions on how I can add one. Else, I will remove this variable.